Automation/FunctionAppUsersUserID/script.ps1

```diff
@@ -0,0 +1,36 @@
+param(
+    [Parameter(Mandatory=$true)] 
+    [string]
+    $TenantID,
+
+    [Parameter(Mandatory=$true)] 
+    [string]
+    $SPApplicationId,
+
+    [Parameter(Mandatory=$true)] 
+    [string]
+    $SPSecret,
+
+    [Parameter(Mandatory=$true)] 
+    [string]
+    $SubscriptionId,
+
+    [Parameter(Mandatory=$true)] 
+    [string] 
+    $resourceGroupName,
+
+    [Parameter(Mandatory=$true)] 
+    [string]
+    $location,
+
+    [Parameter(Mandatory=$true)] 
+    [string]
+    $templateUri
```

Comment:
This should be "URL" not "URI"

Reply:
Yeah I'll look into the names of the resources, and whoops thanks for the heads up! I'll have that changed!